### DISCUSSION TITLE
Can't get sql format to work

### DISCUSSION THREAD

**Discussion by zhoujia74** [diffplug/spotless#1915] [Q&A]
I'm able to use dbeaver to format sql in spotless. But dbeaver doesn't format back tick correctly. It formats a column name like `a-b` to `a - b` (adding extra whitespace) which makes the sql invalid. I don't find a way to fix this issue. So I want to try prettier instead.
I tried the following configs but it doesn't work. I got the error "Caused by: com.diffplug.spotless.npm.SimpleRestClient$SimpleRestResponseException: Unexpected response status code at /prettier/format [HTTP 500] -- (Error while formatting: ConfigError: Couldn't resolve parser "sql".)"

`spotless {
    sql {
        target 'src/main/resources/*.sql'

        prettier(["prettier": "3.0.3", "prettier-plugin-sql": "0.17.0"]).config(['parser': 'sql'])
    }
}`

I tried to google an example to use prettier for sql formatter but can't find one. Can someone point me to a valid example or show me how to use prettier for sql?

**Comment by l3r8yJ** [comment]
@nedtwigg Hi! I occurred the same issue, can you take a look please?
